test/parallel/test-tls-check-server-identity.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var util = require('util');
+const common = require('../common');
+const assert = require('assert');
```

Comment:
I tried removing the block
```if (!common.hasCrypto) {
  common.skip('missing crypto');
  return;
}```
but mean common is not used `error  'common' is assigned a value but never used  no-unused-vars`.
Then when I remove the require for common I get `error  Mandatory module "common" must be loaded  required-modules`